src/NuGetGallery.Core/GitHub/NuGetPackageGitHubInformation.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace NuGetGallery.GitHub
+{
+    public class NuGetPackageGitHubInformation
+    {
+        public NuGetPackageGitHubInformation() : this(0, null)
+        { }
+
+        public NuGetPackageGitHubInformation(int totalRepos, IReadOnlyList<RepositoryInformation> repos)
+        {
+            TotalRepos = totalRepos;
```

Comment:
Please add null checks to all constructors. Also, I would consider ensuring that `totalRepos` is within a logical range (>0)

Reply:
totalRepos can be 0 if a package is not used by any "popular" GitHub repos

Reply:
Ah right. `>=0` then 😄